site/content/blog/argocd-carvel-plugin.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Continuous delivery using a Carvel ytt Argo CD plugin"
+slug: argocd-carvel-plugin
+date: 2022-02-02
+author: Cari Lynn
+excerpt: "Use your favorite Carvel templating tool in your GitOps continuous delivery using the Carvel ytt Argo CD plugin."
+image: /img/logo.svg
+tags: ['ytt', 'argocd', 'gitops']
+---
+
+Argo CD is a declarative, GitOps, continuous delivery tool for kubernetes. It's design follows the GitOps philosophy of using Git as a single source of truth for the desired state of the system. In this example we're storing desired application state in ytt templates, and extending Argo CD to template and deploy them.
```

Comment:
kubernetes => Kubernetes
"It's design follows the GitOps philosophy" => It embraces GitOps philosophy"